test/api/SettingTest.ts

```diff
@@ -211,5 +248,252 @@ describe('Setting tests', function() {
 
       const clientSecretAfter = _.get(readSettingAfter.data.result[0], readSettingAfter.data.result[0].sensitiveData[0]);
     });
+    it('Check crypto settings update - change key + block size (256->128)', async () => {
+      // Retrieve the crypto setting id
+      const read = await getCurrentCryptoData();
```

Comment:
I think you can rename 'read' to something more meaningful like cryptoData, currentCryptoData or similar after the rename of `getCurrentCryptoData()` ;)